Website/Providers/DataProviders/SqlDataProvider/DotNetNuke.Data.SqlDataProvider

```diff
@@ -269,8 +200,17 @@ INSERT INTO {databaseOwner}[{objectQualifier}ContentTypes] ([ContentTypeID], [Co
 INSERT INTO {databaseOwner}[{objectQualifier}ContentTypes] ([ContentTypeID], [ContentType]) VALUES (5, N'DNNCorp_JournalGroup')
 INSERT INTO {databaseOwner}[{objectQualifier}ContentTypes] ([ContentTypeID], [ContentType]) VALUES (6, N'File')
 SET IDENTITY_INSERT {databaseOwner}[{objectQualifier}ContentTypes] OFF
+
 GO
 
+-- minimum date to set for all creted on last modified on columns

```

Comment:
I am asking myself, what this comment means…

Reply:
I would prefer to set these value only once on top of the script - there is a high risk of overlooking one of the occurrences. Would it be possible to restructure the script to have these values first before the first GO? Otherwise we should improve our system of DNN placeholders for SQL and provide it as additional variables for substitution e.g. {DefaultCreatedBy} etc.

Reply:
Well that's what I explained in my previous comment. These vars were originally defined at the beginning of the script, but that worked because there were no GO statements. I tried temp tables also, but didn't work either because temp tables don't survive the connection.

Reply:
> I am asking myself, what this comment means…

Agreed. Removed